modules/tensor_mechanics/src/actions/GeneralizedPlaneStrainAction.C

```diff
@@ -41,12 +41,10 @@ GeneralizedPlaneStrainAction::act()
 {
   if (_current_task == "add_kernel")
   {
-    std::string k_type("GeneralizedPlaneStrainOffDiag");
+    std::string k_type = "GeneralizedPlaneStrainOffDiag";
     InputParameters params = _factory.getValidParams(k_type);
-
     params.set<std::vector<NonlinearVariableName> >("displacements") = _displacements;
-
-    params.set<std::vector<VariableName> >("scalar_strain_zz") = {_scalar_strain_zz};
+    params.set<std::vector<VariableName> >("scalar_strain_zz") = { _scalar_strain_zz };
```

Comment:
Not to rain on my own parade but some of these styles are going to be dictated by clang-format. @rwcarlsen is currently looking into this specifically and it looks like clang-format removes those spaces 😢 .